### DISCUSSION TITLE
v1.22.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#3391] [Releases & Announcements]
## Changelog

- Chatwoot commands bar for improved productivity 
- Ability to define and add custom attributes to conversations/contacts
- Ability to add notes to contact on the CRM page
- Ability to configure Dialogflow integration across all channel types except email
- Ability to rearrange the conversation sidebar
- Ability to specify subject to outgoing emails
- APIs to filter conversations & contacts
- Ability to enable Pre Chat form before triggering campaigns
- Installations will automatically generate vapid keys for push notifications
- Ability to ensure HMAC mandatory for widget interactions
- Numerous bug fixes and enhancements

Thanks to @lybrus, @jdsalchow, @akhilgkrishnan, @virajbahulkar, @jayeclark, @SanjuSudheerM, @sarhugo, @alexverde, @wsuff, @santhosh-chinnasamy, @sarzynski, @Jnig for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v1.22.0'>v1.22.0</a>.</em>

**Comment by Stephen-Lobo** [comment]
Thank you Chatwoot team for new release. 

We are finding difficulty to use new feature - **APIs to filter conversations & contacts**. The API documentation is very brief to understand. 

Just for an example, suppose I want to search/filter Person name 'John' in the contact list. I have tried following query, but it is not working. 

```
{
    "name": "FindJohn",
    "type": "contact",
    "query": {
                "attribute_key": "contact",
                "attribute_name": "John",
                "input_type": "plain_text",
                "data_type": "text",
                "filter_operators": {"equal_to"},
                "attribute_type": "standard"
            }
}
```

If possible, please provide PostMan example for this POST query.

Regards

**Reply by Stephen-Lobo** [reply]
@sojan-official  / @pranavrajs 

Waiting for reply.

**Reply by sojan-official** [reply]
@Stephen-Lobo this API is still evolving as we are finalizing the filters. So there could be minor changes. I would recommend waiting until the feature is fully shipped in our next release. 

cc: @tejaswinichile
